android/expoview/src/main/java/versioned/host/exp/exponent/modules/internal/DevMenuModule.kt

```diff
@@ -223,5 +233,29 @@ class DevMenuModule(reactContext: ReactApplicationContext, val experiencePropert
     return reactApplicationContext.resources.getString(ref)
   }
 
+  /**
+   * Indicates whether the underlying js executor supports inspecting.
+   * NOTE: because current react-native doesn't pass jsi runtime `isInspectable` to java,
+   * workaround to determine the state by executor name.
+   */
+  private val isJsExecutorInspectable: Boolean by lazy {
+    val activity = currentActivity as? ReactNativeActivity?
```

Comment:
```suggestion
    val activity = currentActivity as? ReactNativeActivity
```
With a safe cast, the delivered type is always nullable.